test/cql-pytest/test_tablets.py

```diff
@@ -201,3 +201,16 @@ def test_tablets_are_dropped_when_dropping_index(cql, test_keyspace, drop_index)
         except:
             pass
         raise e
+
+
+# Test that a LWT query fails with a tablets-enabled table.
+# See issue #18068
+def test_lwt_support_with_tablets(cql, test_keyspace, skip_without_tablets):
+    with new_test_table(cql, test_keyspace, "key int PRIMARY KEY, val int") as table:
+        cql.execute(f"INSERT INTO {table} (key, val) VALUES(1, 0)")
+        with pytest.raises(InvalidRequest, match=f"{table}.*LWT is not yet supported with tablets"):
+            cql.execute(f"INSERT INTO {table} (key, val) VALUES(1, 1) IF NOT EXISTS")
```

Comment:
Please also check the prepared statement case. If I understand correctly, the prepare() will (unfortunately) work - but then the execute() will fail with InvalidRequest (which is very strange, so let's verify that it actually works).

Reply:
apologies, I missed it

Reply:
added in v6